griptape/engines/extraction/base_extraction_engine.py

```diff
@@ -35,13 +37,13 @@ def validate_max_token_multiplier(self, _: Attribute, max_token_multiplier: int)
 
     @property
     def max_chunker_tokens(self) -> int:
-        return round(self.prompt_driver.tokenizer.max_input_tokens * self.max_token_multiplier)
+        return round(self.prompt_engine.prompt_driver.tokenizer.max_input_tokens * self.max_token_multiplier)
```

Comment:
Add a convenient `tokenizer` property method to `PromptEngine` for retrieving the tokenizer, since this access pattern is somewhat common.

In general, it'd be best to encapsulate the implementation details of `PromptEngine` as much as possible. Ideally, we should never be calling methods on `prompt_engine.prompt_driver`.

Reply:
done